sample/kotlin/build.gradle.kts

```diff
@@ -195,6 +195,7 @@ dependencies {
     implementation("io.opentracing.contrib:opentracing-rxjava-3:0.1.4") {
         exclude(group = "io.opentracing")
     }
+    implementation("com.launchdarkly:okhttp-eventsource:2.5.0")
```

Comment:
it is already included in the `// Network` section below

Reply:
👍